int8 followerID
---
int8 leaderID